src/horus_api.h

```diff
@@ -35,20 +35,75 @@
 #include <stdint.h>
 #include "modem_stats.h"
       
-#define HORUS_MODE_BINARY            0
-#define HORUS_MODE_RTTY              1
+/* Horus API Modes */
+#define HORUS_MODE_BINARY            0  // Legacy binary mode
+#define HORUS_MODE_BINARY_256BIT     1  // New 256-bit LDPC-encoded mode
+#define HORUS_MODE_BINARY_128BIT     2  // New 128-bit LDPC-encoded mode
+#define HORUS_MODE_RTTY              99 // RTTY Decoding
+
+
+// Settings for Legacy Horus Binary Mode (Golay Encoding)
+#define HORUS_BINARY_NUM_BITS                   360   /* fixed number of coded bits */
+#define HORUS_BINARY_NUM_PAYLOAD_BYTES          22    /* fixed number of bytes in binary payload     */
+#define HORUS_BINARY_DEFAULT_BAUD               100   /* Default baud rate of 100 baud */
+#define HORUS_BINARY_DEFAULT_TONE_SPACING       270   /* Default tone spacing of 270 Hz */
+
+// Settings for Horus Binary 256-bit mode (LDPC Encoding, r=1/3)
+#define HORUS_BINARY_256BIT_NUM_BITS            768
+#define HORUS_BINARY_256BIT_NUM_PAYLOAD_BYTES   32
+#define HORUS_BINARY_256BIT_DEFAULT_BAUD        100   /* Default baud rate of 100 baud */
+#define HORUS_BINARY_256BIT_DEFAULT_TONE_SPACING 270   /* Default tone spacing of 270 Hz */
```

Comment:
We have Rs around 25Hz in https://github.com/drowe67/codec2/pull/117

Reply:
I figure this is all subject to change. I would like to make use of the larger frame size at a higher baud rate as well as the lower rate. This would mean improved performance and expanded functionality while retaining the current update rate. I guess the selection of the 'robust' version (e.g. lower baud rate) could be somewhere else? Apart from the baud rate all the other settings will be the same.

Reply:
Fine.  But  the reason I spent a few weeks in re-factoring and improving the FSK modem -  is the low SNR/new protocol aspect - and that's where I'd like to focus this v2/3 protocols.  So can we make your different update rates an option - and default to previously agreed goals of the project?